apps/reference/nav/_referenceSidebars.js

```diff
@@ -173,7 +173,13 @@ const sidebars = {
         'guides/realtime',
         'guides/realtime/quickstart',
         'guides/realtime/postgres-cdc',
-      ],
+          {
+            type: 'doc',
+            id: 'guides/realtime/limits', // document ID
+            label: 'Limits', // sidebar label
```

Comment:
```suggestion
            label: 'Rate Limits', // sidebar label
```

Reply:
Realized I could do a `sidebar_label` in the doc!  This is better now.